geode-concurrency-test/src/main/java/org/apache/geode/test/concurrency/RunnableWithException.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.test.concurrency;
+
+import java.io.Serializable;
+
+/**
+ * Like Runnable, but allows the run method to throw an exception
+ */
+public interface RunnableWithException extends Serializable {
```

Comment:
What does this interface gain us that `Callable` doesn't cover?

Reply:
It means you don't have to return a value. So you can do

executor.inParallel(() -> System.out.println("hi"))

instead of 

executor.inParallel(() -> { 
  System.out.println("hi");
  return null;
)